test/unit/spec/VerifyPIV_spec.js

```diff
@@ -97,8 +101,9 @@ function (Q, Okta, OktaAuth, Util, PivForm, PrimaryAuthForm, Beacon, Expect,
       });
     });
     Expect.describe('Error', function () {
+      var pivError = deepClone(resError);
```

Comment:
nit: you probably don't need this, you can pass `resError` in the tests that do not change anything in that object

Reply:
Because of this logic - https://github.com/okta/okta-signin-widget/blob/master/test/unit/helpers/mocks/Util.js#L104-L107 I've been having issues with these tests because it modifies the response directly.. One way around this was to do the above

Reply:
not sure I follow why, maybe we can talk offline. Doesn't hurt to clone the response here, I just want to know whats going on

Reply:
talked offline, thanks for explaining what's going on.
Util changes the original object to add a `responseText`, so after the first run we wouldn't be able to test a response that has no `responseText`.